### PR TITLE
Tokens

### PR DESCRIPTION
**Pull request by alexandergeeyudu**
@jrdh 

Updates the doc to include the new token resource.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -849,6 +851,101 @@ The authentication resource allows a user to authenticate a given reader's passw
...
+A simple use-case could be as follows:
+
+1. A reader clicks on a link on your webpage indicating they wish to view an edition.
+2. Your server reacts to that request by:
+    1. sending a request to this API for a token for that user
+    2. retrieving the token value from the response
+    3. inserting the token value and user ID into the edition's target URL
+    4. returning a 303 redirect with the modified edition URI as the target
+3. The reader's browser redirects to the edition and the edition uses the token to authenticate.
+
```
Comment (jrdh) [inline README.md]: @tobiaspowell happy with this as a brief guide?

Comment (jrdh) [inline README.md]: (I am)

**Comment by jrdh** [comment]
Looks good to me :+1:

File: README.md (hunk 1 above)
Comment (tobiaspowell) [inline README.md]: Think so. It should be fine for anyone with good developers, and for anyone who needs more than this we'll probably need an entire reference implementation - which can wait until we have to create one.

**Pull request merged**